templates/otp.conf.j2

```diff
@@ -0,0 +1,9 @@
+description "start OpenTripPlanner process"
+
+start on {{ upstart_start_on }}
+stop on runlevel [!2345]
+
+chdir {{ otp_bin_dir }}
+script
+    exec sudo -u {{ otp_user }} ./otp -a --graphs {{ otp_data_dir }} --server
```

Comment:
@flibbertigibbet good point.

Assuming -Xmx is max jvm memory. Is there a good default value for that, in your experience? 30Gb is really high...